extension/src/main/java/com/mobven/extension/View.kt

```diff
@@ -127,6 +127,46 @@ fun NestedScrollView.setInsetListener() {
     }
 }
 
+/**
+ * Extension method for callback action on tab reselected
+ */
+fun TabLayout.doOnTabReselected(onReselected: (TabLayout.Tab?) -> Unit) {
```

Comment:
TabLayout'un her bir extension'u için ayrı ayrı listener'lar ekleniyor bu şekilde. Bunu şöyle yapabiliriz. TabLayout.onTabSelectedListener({}, {}, {}) tek bir extension yapalım.

Reply:
Butun tab listener extensionlari tek bir extension functionda birlestirildi